src/main/java/gregtech/api/recipes/chance/ChanceEntry.java

```diff
@@ -0,0 +1,26 @@
+package gregtech.api.recipes.chance;
+
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * An entry for an ingredient and associated chance of operation.
+ *
+ * @param <T> the type of ingredient contained by the chanced entry
+ */
+public interface ChanceEntry<T> {
```

Comment:
should this not be called `IChanceEntry`?

Reply:
I think we should move away from the dated `I`-prefixed naming scheme for interfaces. We've been slowly doing so over the past few months, so I think it's fine.

Originally this was an `@FunctionalInterface` which would have made it more awkward to understand as one, when `I`-prefixed.

Reply:
ah